consumer/main.go

```diff
@@ -49,7 +27,14 @@ func main() {
 	taskQueue := redisConn.OpenQueue(common.RedisQueueName)
 	taskQueue.StartConsuming(100, time.Second)
 
-	taskConsumer := &taskConsumer{}
+	db, err := datastore.NewDatastore(datastore.REDIS, dbConnectionString)
+	if err != nil {
+		log.Print(err)
+	}
+	defer db.Close()
+
+	env := datastore.Env{DB: db}
+	taskConsumer := &consumer.TaskConsumer{DbEnv: &env}
 	taskQueue.AddConsumer("task consumer", taskConsumer)
```

Comment:
i missed this can you move this to the common constants file and rename it to `tweet consumer`